core/service-test-runner/cli.js

```diff
@@ -15,6 +15,11 @@
 // Run tests on a given instance:
 //   SKIP_INTERCEPTED=TRUE TESTED_SERVER_URL=https://test.shields.io npm run test:services --
 //
+// Run tests with given number of retries and backoff:
+//   RETRY_COUNT=3 RETRY_BACKOFF=100 npm run test:services --
```

Comment:
Shall we indicate that it's in milliseconds here?

Reply:
Thanks. Done.